game/entities.py

```
@@ -0,0 +1,252 @@
+            max_change = 1
+            
+        # Apply random variations
+        for i in range(1, self.width - 1):
+            # Don't change the edges
+            change = random.randint(-max_change, max_change)
+            self.heights[i] = max(1, min(self.height - 1, self.heights[i] + change))
+            
+            # Smooth it out a bit with its neighbors
+            self.heights[i] = (self.heights[i - 1] + self.heights[i] * 2 + self.heights[i + 1]) // 4
+    
+    def _create_landing_zones(self):
```

Ruff (0.8.2)

`_create_landing_zones` is too complex (13 > 10)

(C901)